src/Sql/Sql/Auditing/Services/AuditingEndpointsCommunicator.cs

```diff
@@ -166,6 +166,11 @@ public AuditingEndpointsCommunicator(IAzureContext context)
                 }
             }
 
+            if (settings.Logs.FirstOrDefault(l => string.Equals(l.Category, DefinitionsCommon.SQLSecurityAuditCategory)) == null)
```

Comment:
@bashahee minor refactor you can do to the if-statement:

```cs
if (!settings.Logs.Any(l => string.Equals(l.Category, DefinitionsCommon.SQLSecurityAuditCategory))
{
    settings.Logs.Add(new LogSettings(true, DefinitionsCommon.SQLSecurityAuditCategory));
}
```

Reply:
Done